prime-dotnet-webapi/Migrations/20200929200820_AgreementRenameandAddOrgAndParty.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Prime.Migrations
+{
+    public partial class AgreementRenameandAddOrgAndParty : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_AccessTerm_Agreement_AgreementId",
+                table: "AccessTerm");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_AccessTerm_Enrollee_EnrolleeId",
+                table: "AccessTerm");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_AccessTerm_LimitsConditionsClause_LimitsConditionsClauseId",
+                table: "AccessTerm");
+
+            migrationBuilder.DropIndex(
+                name: "IX_AccessTerm_AgreementId",
+                table: "AccessTerm");
+
+            migrationBuilder.DropIndex(
+            name: "IX_AccessTerm_EnrolleeId",
+            table: "AccessTerm");
+
+            migrationBuilder.DropIndex(
+                name: "IX_AccessTerm_LimitsConditionsClauseId",
+                table: "AccessTerm");
+
+            migrationBuilder.RenameTable(
+                name: "Agreement",
+                schema: null,
+                newName: "AgreementVersion");
+
+            migrationBuilder.RenameTable(
+                name: "AccessTerm",
+                schema: null,
+                newName: "Agreement");
+
+            migrationBuilder.RenameColumn(
+                name: "AgreementId",
+                table: "Agreement",
```

Comment:
I think this is incorrect, if the migrations work the way I think they do. This is modifying the new Agreement table (the one previously called AccessTerms), rather than the old Agreement table (the one now called AgreementVersion). this line should be `table: "AgreementVersion`

Reply:
This was previously called AgreementId on the AccessTerm table, now the AccessTerm table is called Agreement and AgreementId is actually AgreementVersionId, this links Agreement to AgreementVersion. This should be updating the new Agreement table. Renaming a table to one that already existed before is confusing haha